scripts/l2-transfer.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * This script transfers ETH between Optimistic Mainnet accounts
+ */
+
+const { providers, Wallet, utils, Contract } = require("ethers");
+const { getContractInterface } = require("@eth-optimism/contracts");
+const { JsonRpcProvider } = providers;
+
+const cfg = config();
+
+const provider = new JsonRpcProvider("https://mainnet.optimism.io");
+const wallet = new Wallet(cfg.privateKey).connect(provider);
+const l2ETHContract = new Contract(
+  "0x4200000000000000000000000000000000000006",
+  getContractInterface("OVM_ETH"),
+  wallet
+);
+
+(async () => {
+  const address = await wallet.getAddress();
+  console.log(`Sending from ${address}`);
+
+  const balance = await wallet.getBalance();
+  console.log(`Balance: ${utils.formatEther(balance.toString())}`);
+
+  console.log(`Sending to ${cfg.to}`);
+
+  const tx = await l2ETHContract.transfer(cfg.to, utils.parseUnits(cfg.amount));
```

Comment:
You should be able to do this with normal transactions:

`await wallet.sendTransaction({ to: cfg.to, value: ... })`

Reply:
That way resulted in "cannot estimate gas" error. I'm assuming I need to specify the gas limit? 
Is there a strong reason to do it that way vs instantiating the contract?